question15/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link href="style.css" rel="stylesheet">
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <input type="button" id="js-modal-btn" value="button">
+        <div id="js-mask" class="hidden"></div>
+        <div id="js-modal" class="hidden">
+            <div id="js-close">×</div>
+            <div class="inputAreaContainer">
+                <div class="inputArea">
+                    <p>ID</p>
+                    <input type="number" id="js-number" placeholder="数値を入力してください">
+                </div>
+                <div class="inputArea">
+                    <p>氏名</p>
+                    <input type="text" id="js-text" placeholder="文字を入力してください">
+                </div>
+            </div>
+            <input type="submit" id="js-request-btn" value="送信">
+        </div>
```

Comment:
Since this is a block that sends data, I thought it would be better to enclose it in a form tag.
Also, I think it would be better to use label tags for things like "ID" that are tied to input elements.

https://code-kitchen.dev/html/input/#%E5%9F%BA%E6%9C%AC%E7%9A%84%E3%81%AB%E3%83%95%E3%82%A9%E3%83%BC%E3%83%A0%EF%BC%88form%EF%BC%89%E3%81%AE%E4%B8%AD%E3%81%A7%E4%BD%BF%E7%94%A8%E3%81%99%E3%82%8B

Reply:
Thanks for pointing that out! I've learned a lot and added the form and label tags. Please confirm.Thank you for the clear article. It is very helpful!! 
latest commit: https://github.com/csakyo/homework/pull/28/commits/e6b2e09bbcc466555c04e0d0197fa242894556e8